lib/src/public_api_parser.dart

```diff
@@ -52,6 +52,9 @@ class PublicApiParser {
     } else if (map.containsKey('insertionOrders')) {
       return List.from(map['insertionOrders'])
           .map((ioMap) => _createInsertionOrder(ioMap))
+          .where((io) =>
```

Comment:
what does this do?

Reply:
This filters out the IOs that are not in-flight when the query type is byAdvertiser/byCampaign. If the query type is byInsertionOrder, then we don't do any filter. And if that single IO requested is not in-flight, the budget/spending/start date/end date will be empty.